---
string message
float64 gripper_angle
